exercises/hello_rubyric/config.yaml

```diff
@@ -0,0 +1,17 @@
+
+_external: true
+title: Hello Rubyric
+description: A simple grading example with Rubyric and Python
+max_submissions: 10
+max_points: 10
+points_to_pass: 6
+min_group_size: 1
+max_group_size: 2
+key: rubyric-python
+lti: Rubyric
+lti_context_id: ''
```

Comment:
This doesn't seem nice, I think this removes the uniquiness of the exercise in LTI protocol. I might be wrong of course.

Reply:
What exactly does not seem right?

Reply:
This should set the `lti_context_id` to an empty string. LTI context id is used to differentiate exercises in the tool provider side. Thus, two exercises with the same empty string could not be differentiated by it.